sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -73,11 +73,11 @@ cognitive services.
     ```
     
     Use the key as the credential parameter to authenticate the client:
-    <!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L45-L48 -->
+    <!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L44-L47 -->
     ```java
     TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
-        .subscriptionKey(SUBSCRIPTION_KEY)
-        .endpoint(ENDPOINT)
+        .subscriptionKey(new TextAnalyticsSubscriptionKeyCredential("{subscription_key}"))
+        .endpoint("https://{servicename}.cognitiveservices.azure.com/")
```

Comment:
maybe we should leave it as before `ENDPOINT`? The customer can go to the portal or with Azure CLI and get the endpoint.
Leaving it like this, might confuse people when using a multi-service subscription key as the region goes before the endpoint

Reply:
Good to know this. I referred to the docs we have. I found nowhere has an example of multi-service key endpoint. Can you share an example of it?

Reply:
sure, see here: https://docs.microsoft.com/en-us/azure/cognitive-services/authentication#authenticate-with-a-multi-service-subscription-key

Reply:
Thank you